labels.mss

```diff
@@ -491,26 +491,13 @@
      designated/marker-max-error: 0.5;
      designated/marker-spacing: 50;
      designated/marker-fill: @cycle-fill;
-     designated/marker-file: url(symbols/oneway.svg);
+     designated/marker-file: url(symbols/nounproject/sharrows.svg);
      [oneway='-1'] { designated/marker-file: url(symbols/oneway-reverse.svg); }
      [zoom=15] {
         designated/marker-transform: "scale(0.75)";
         designated/marker-spacing: 40;
      }
   }
-  [bicycle='designated'][oneway = 'yes'][oneway_bicycle = 'no'],
```

Comment:
I think this is a duplicate with https://github.com/cyclosm/cyclosm-cartocss-style/pull/542/files#diff-7f4b9a4751a0a11cd2e9b3a1919623c344185a0cbce66d3534b4ea26066fcffaR473-R485. But to be double checked.

Reply:
Duplicate of what ?
This part is for designated on a road cyclable in both direction.
Here we need a both direction sharrow like : <<>>